src/index.js

```diff
@@ -84,6 +91,14 @@ export default class DragDrop {
           blockContent.style.removeProperty('border-bottom');
           this.endBlock = this.getTargetPosition(dropTarget);
           this.moveBlocks();
+
+          // If the item changes something after drop it will replace the dropped item with the original
+          if (!this.holder.contains(this.itemDragged)) {
+            if (this.holder.children[0].children[0].children[this.endBlock]) {
+              const itemDropped = this.holder.children[0].children[0].children[this.endBlock].children[0];
+              itemDropped.replaceWith(this.itemDragged);
+            }
+          }
```

Comment:
@santiagoboterokommit would make sense to abstract this code fragment into a function? What do you think?